src/integrators/sdirk_integrators.jl

```diff
@@ -445,7 +430,7 @@ end
 
   cache.ηold = η
   cache.newton_iters = iter
-  @. u = uprev + 2*z
+  @. u += z + dz
```

Comment:
Mathematically it should be correct, I guess, but again it is apparently not a good idea because it could accumulate floating point errors.